packages/app/cypress/e2e/runner/snapshots/runner.mochaEvents.cy.ts.json

```diff
@@ -1004,7 +1009,8 @@
         "_cypressTestStatusInfo": {
           "shouldAttemptsContinue": false,
           "attempts": 1,
-          "outerStatus": "failed"
+          "outerStatus": "failed",
+          "reasonToStop": "PASSED_FIRST_ATTEMPT"
```

Comment:
if the `outerStatus` is failed then how come the reasonToStop is `PASSED_FIRST_ATTEMPT`?

Reply:
Thanks for catching this! I fixed it.